source/DSCResources/DotNetFramework/DotNetFramework.schema.psm1

```diff
@@ -60,12 +60,18 @@ configuration DotNetFramework
         [Parameter()]
         [ValidateNotNullOrEmpty()]
         [string[]]
-        $SkipRuleType
+        $SkipRuleType,
+
+        [Parameter()]
+        [ValidateNotNullOrEmpty()]
```

Comment:
Being that there is a ValidateSet attribute is the ValidateNotNullOrEmpty attribute even needed since NULL is not in the validateSet?

Reply:
Agreed.